### ISSUE TITLE
Please update installation instructions for FreeBSD

### ISSUE THREAD

**Issue by yurivict** [dartsim/dartsim.github.io#20]
In order to install from source on FreeBSD, this command should be run as root:
```
cd /usr/ports/misc/dartsim && pkg install -A $(make missing) && make install clean
```

There is no need to list dependencies.
Regarding the missing port ```pagmo```, I will create it withing a day. There is no need to list it separately either.

https://github.com/dartsim/dartsim.github.io/pull/19/commits/47f0feb0ad149005b40835ef668236a4e31cc19d

**Comment by jslee02** [comment]
Thanks for the guide and working on `pagmo` port! https://github.com/dartsim/dartsim.github.io/pull/21 is created.

**Comment by jslee02** [comment]
Resolved by #21
